modules/master-node-sizing.adoc

```diff
@@ -6,21 +6,36 @@
 [id="master-node-sizing_{context}"]
 =  Control plane node sizing
 
-The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing.
-
-[options="header",cols="3*"]
+The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing. The control plane tests create the following objects across the cluster in each of the namespaces depending on the node counts:
+
+----
+12 imagestreams
+3 buildconfigs
+6 builds
+1 deployment with 2 pod replicas (sleep) mounting two secrets each. deployment-2pod
+2 deployments with 1 pod replicas (sleep) mounting two secrets. deployment-1pod
```

Comment:
what does sleep mean here? not sure this will translate well, so we should consider rewording
also, what is the significance of the deployment-2pod and deployment-1pod? Is that the name of each of these items?

Reply:
Sleep here means that the pods are just idle, rephrased it to skip the sleep to avoid any confusion. Yes deployment-2pod and deployment-1pod are the names of the two items.